packages/fxa-content-server/tests/functional/lib/helpers.js

```diff
@@ -2890,3 +2890,17 @@ module.exports = {
   visibleByQSA,
   waitForUrl,
 };
+
+// Export helpers methods in a form that can be easily used in
+// async/await. The usage of the helpers are the same except
+// they expect the last argument to be the intern remote object.
+const helpersRemoteWrapped = {};
+const helpers = { ...module.exports };
+Object.keys(helpers).forEach((key) => {
+  helpersRemoteWrapped[key] = async function () {
+    const args = [...arguments];
+    const remote = args.pop();
+    return remote.then(helpers[key](...args));
+  };
+});
```

Comment:
nit: copying `module.exports` to `helpers` doesn't seem necessary.

Reply:
oops sorry, I wasn't very clear with my comment.  Couldn't we use `module.exports` instead of making a copy with `{...module.exports}`?